config/rbac/submariner-diagnose/cluster_role.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: submariner-diagnose
+rules:
+  # submariner-diagnose runs subctl diagnose to troubleshoot submariner
+  # dpeloyment
+  - apiGroups:  # nodes are looked up to figure out network settings
+      - ""
+    resources:
+      - configmaps
+      - nodes
+    verbs:
+      - get
+      - list
+  - apiGroups:  # pods are created to run firewall diagnostics
+      - ""
+    resources:
+      - pods
+    verbs:
+      - create
```

Comment:
Is it necessary to create pods on the `ClusterRole`? You already grant create pods on the `Role`

Reply:
Yes. Firewall test creates pods in "default" namespace. Permissions in `Role` aren't enough.

Reply:
I wonder if its possible to bind the existing `Role` to the `default` namespace instead of granting cluster wide `pod create` permissions?

Reply:
That should be possible, yes.

Reply:
https://github.com/submariner-io/submariner-operator/issues/1885

Reply:
Is it possible to bind role to two namespaces? We also need pod creation in `submariner-operator` namespace to allow Job to run the pods. It was failing for me without pod create permission in the role. Or create two roles? Alternately, why not modify `diagnose` to create pods in operator namespace?

Reply:
No, a role is a namespaced object; so you’d have to create it in all namespaces where it’s needed, and bind the SA to all the corresponding roles. We could use a single namespace for everything instead.

Reply:
So it should be possible to create the `Role` and `RoleBinding` for that in the default namespace, or just use the single namespace for diagnosis (which makes sense anyhow I guess).